layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -1,10 +1,33 @@
 UBOOT_KCONFIG_SUPPORT = "1"
 inherit resin-u-boot
 
```

Comment:
If you are not using u-boot-ti-staging then all these can be removed

Reply:
I agree - these changes can be removed - will update the PR accordingly.